test/regression/expected/basic.out

```diff
@@ -81,3 +81,23 @@ SELECT * FROM t JOIN s ON a = b;
 
 DROP TABLE t;
 DROP TABLE s;
+CREATE TABLE a  (b int);
+CREATE TABLE "A"(c int);
+INSERT INTO a VALUES(1);
+SET duckdb.force_execution = false;
+SELECT * FROM a, "A" as aa;
+ b | c 
+---+---
+(0 rows)
+
+SET duckdb.force_execution = true;
+SELECT * FROM a, "A" as aa;
+WARNING:  (PGDuckDB/CreatePlan) Prepared query returned an error: 'Binder Error: Table "aa" does not have a column named "c"
```

Comment:
So, the error is still there right?

Reply:
Yes, there will be this warning, because DuckDB failed to run, so it was run by PG, which makes the result ultimately consistent with PG. If we don't want to see this warning here, maybe we need to wait for #477 as you said.